src/styles/Carousel.scss

```diff
@@ -1,29 +1,33 @@
-.BrainhubCarousel {
-  overflow: hidden;
-  display: flex;
-  align-items: center;
+.BrainhubCarousel__container {
+  width: 100%;
```

Comment:
this solution will not work well with `flexFlow: row`

Reply:
I'll look into this

Reply:
@humbak This seems to work fine with flex-flow row, see: https://imgur.com/a/dEFQE7k

Reply:
@max-arias use more text and you will see something like it: 
![image](https://user-images.githubusercontent.com/26634399/81556753-25139900-938b-11ea-91bd-2cba294a2298.png)

Reply:
Thanks, updated. Let me know.